### ISSUE TITLE
not-so-BigGAN: Generating High-Fidelity Images on a Small Compute Budget

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#112]
# TL;DR
This is a research to generate a high-fidelity image using GANs with a small amount of computational resources by reproducing the image from the low-frequency information only using the Wavelet Transform (WT). Specifically, the lost high-frequency information is reconstructed by NN and the inverse Wavelet Transform (iWT) is applied. Although the quality of the generated images decreases a little, they have succeeded in reducing the computational resources from TPUx256 to GPUx4.

![= Learned](https://user-images.githubusercontent.com/24524018/93408824-751de180-f8d0-11ea-8271-f23733757545.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.04433

# Submission Dates(yyyy/mm/dd)
2020/09/09

# Authors and institutions
Seungwook Han, Akash Srivastava, Cole Hurwitz, Prasanna Sattigeri, David D. Cox
- IBM Research
- MIT-IBM Watson AI Lab
- University of Edinburgh

# Methods

# Results
![(a) NOT-SO-BIGGAN-PIXEL](https://user-images.githubusercontent.com/24524018/93408832-7949ff00-f8d0-11ea-9016-cb2eb912f876.png)

# Comments
